docs/Relay Release Engineer playbook.md

```diff
@@ -0,0 +1,48 @@
+## Relay: Release Engineer playbook
+
+1. Tuesday PM: [Release to stage][Release-to-stage] (tag, Github release notes)
+
+    a. Ping all the engineers who have changes in the release to:
+      * Move cards to “Ready to Test” if necessary
+      * Include instructions for QA to test
+    
+    b. Confirm any hotfixes are also in the new tag
+
+2. Wednesday-Monday
+
+    a. Use the [Sentry Releases][sentry-releases] report to watch for any new issues
+    
+    b. Co-ordinate, re-tag for [stage fixes][stage-fixes] as needed
+
+
+3. Monday
+    
+    a. Release Readiness Review (led by Easy)
```

Comment:
suggestion (blocking): Let's link this to the release readiness review meeting agenda doc. So if that agenda ever changes, we don't have to change it here too.

Reply:
@groovecoder, I don't see a release readiness review meeting agenda doc. Can you share the link?